meinberlin/apps/projects/serializers.py

```diff
@@ -106,6 +99,9 @@ class Meta:
             "url",
         ]
 
+    def get_topics(self, instance):
+        return [topic.code for topic in instance.topics.all()]
```

Comment:
this serializer is meant only to read existing topics of a project? not to add topics to project (write mode)? @goapunk

Reply:
Currently we only read them, or do we have code anywhere which adds them through the serializer?